beetsplug/alternatives.py

```diff
@@ -84,6 +83,16 @@ def parse_args(self, args):
         return self.parser.parse_args(args), []
 
 
+class ArgumentParser(argparse.ArgumentParser):
+    """
+    Facade for ``argparse.ArgumentParser`` so that beets can call
+    `_get_all_options()` to generate shell completion.
+    """
+    def _get_all_options(self):
+        # FIXME return options like ``OptionParser._get_all_options``.
```

Comment:
Maybe we should open an issue to track this?

Reply:
I’ll create one once this PR is merged.